setup.py

```diff
@@ -78,6 +79,8 @@ def get_version(verbose=1, filename='src/qtt/version.py'):
       license='MIT',
       package_dir={'': 'src'},
       packages=find_packages(where='./src'),
+#      package_data={'': ['exampledata/*.*', 'exampledata/*/*.*', 'exampledata/*/*/*.*', 'exampledata/*/*/*/*.*']},
+      include_package_data = True,
```

Comment:
Should this one be set to true when you don't use package_data?

Reply:
Documentation is not clear on this. Leaving it as it was.